### ISSUE TITLE
Naming standard of sudosh vs sudosh2

### ISSUE THREAD

**Issue by Chewbakka-Wakka** [squash/sudosh2#47]
I note this tool is named as sudosh2, yet the resultant built files are named sudosh.

man1/sudosh
man5/sudosh.conf
man8/sudosh-replay

"sudosh2 was forked from sudosh, which was pulled by its original developer when a commercial company bought it from him"

Can all respective files be moved accordingly to match as sudosh2?

**Comment by Chewbakka-Wakka** [comment]
An example seen elsewhere would be bzip2:

$ rpm -qvl bzip2 

/usr/bin/bunzip2 -> bzip2
/usr/bin/bzip2
/usr/bin/bzip2recover
/usr/share/doc/bzip2
/usr/share/man/man1/bzip2.1.gz

**Comment by squash** [comment]
I agree the naming thing is a bit odd but I'll leave it for a few reasons.

The comparison with bzip2 is very different, where the bzip authors released an incompatible recreation of the original bzip, and the two might be installed together on the same host. With sudosh, the projects mostly differentiated the repository where development was happening in a way that tried to make it clear that it was not the original author. At least initially, the differences you would find between the original sudosh project and mine was that mine worked on modern systems and didn't crash in several situations where theirs did, but you wouldn't have both installed at the same time. The sudosh2 project should be a complete replacement, not an added tool. Given that a decade has passed since then and the original project effectively doesn't exist, I would lean towards renaming the repository to sudosh rather than the binary to sudosh2.

I can see a potential future where a real replacement effort exists that would make a binary renaming the clear choice, however that's an undertaking I don't have free cycles for in the short term.

**Comment by WLTBAgent** [comment]
Closing as won't fix. Renaming the binary would break every existing sudoers configuration and session playback setup. Not justified for a cosmetic change.
